src/test/java/com/fasterxml/jackson/core/json/StringGenerationTest.java

```diff
@@ -86,6 +87,17 @@ void longerRandomMultiChunk() throws Exception
         }
     }
 
+    @Test
+    public void testWritingSurrogatePairs() throws IOException {
+        ByteArrayOutputStream stream = new ByteArrayOutputStream();
+        JsonGenerator generator = FACTORY.createGenerator(stream, JsonEncoding.UTF8).enable(Feature.WRITE_UTF8_SURROGATES);
+        String string = "システム\uD867\uDE3D"; // システム𩸽
```

Comment:
Yes, def need to use Unicode escapes for Katakana characters.

Reply:
Test deleted. Surrogate223Test cover the new feature instead.